### DISCUSSION TITLE
How to achieve fuzzy matching and full matching

### DISCUSSION THREAD

**Discussion by Friedrich-hue** [qdrant/qdrant#2732] [Q&A]
There is a piece of text,
The direct conversion of the text into a vector and then querying it has limited effectiveness. I want to implement the following functionality separately:
Is it possible to first break down the question into keywords and then perform a fuzzy matching in the database's payload using Qdrant?

**Comment by generall** [comment]
If is possible to filter by keywords, but fuzzy matching (aka Liechtenstein distance) is not supported at the moment

**Reply by Friedrich-hue** [reply]
Is it sufficient to directly call the interface for text filtering in pyload, or is it necessary to build an index beforehand?

**Reply by generall** [reply]
Without index it will work as a substring match

**Reply by ysdk2** [reply]
@generall Is there a plan to add fuzzy matching anytime in the near future?
